radio/src/datastructs.h

```diff
@@ -125,8 +125,6 @@ static inline void check_struct()
 #elif defined(RADIO_TPRO)
   CHKSIZE(RadioData, 869);
   CHKSIZE(ModelData, 6290);
-#elif defined(RADIO_POCKET)
-  CHKSIZE(RadioData, 869);
```

Comment:
I would suggest this is most definitely **not** the _fix_ as 2.10 branch cpn, etc, compiled last week with this present, and no issues, so something else has changed in this branch. But ModelData is missing. Also, no check for `RADIO_TPROS`?

Reply:
edit: you added the missing `CHKSIZE(ModelData, 6326)` as part of the initial commit... perhaps the value was wrong ;)

Reply:
Sorry to disagree:
https://github.com/EdgeTX/edgetx/blob/bea183989485d4301f169db442fe076dbd33d7b1/radio/src/datastructs.h#L128

Pocket seems wrong there as only RadioData is checked, suggested fixe will default it to:

https://github.com/EdgeTX/edgetx/blob/bea183989485d4301f169db442fe076dbd33d7b1/radio/src/datastructs.h#L145

Reply:
Why sorry? :upside_down_face: As I commented afterwards, it was the addition of ModelData with the wrong value that was the cause of the original build failure, but neither is indeed not necessary if there is no difference with X7 struct. So this is this the case for the TPros S also?

Reply:
yes, same as x7 for TProS (no customisable)